workflow-testing/src/test/java/com/squareup/workflow1/ParameterizedTestRunner.kt

```diff
@@ -0,0 +1,69 @@
+package com.squareup.workflow1
+
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertFalse
+import kotlin.test.assertNotNull
+import kotlin.test.assertNotSame
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+/**
+ * This file is copied from workflow-runtime:commonTest so our tests that test across the runtime
+ * look consistent. We could have used a JUnit library like Jupiter, but didn't.
+ *
+ * This file is copied so as to avoid creating a workflow-core-testing module (for now).
+ *
+ * We do our best to tell you what the parameter was when the failure occured by wrapping
+ * assertions from kotlin.test and injecting our own message.
+ */
+class ParameterizedTestRunner<P : Any> {
```

Comment:
Should we slap the experimental annotation here?

Reply:
Sure. not too concerned either way.